README.md

```diff
@@ -0,0 +1,45 @@
+# Eat Out Log
+
+### Project Status:
+
+##### Build information by branch
+
+master: [![Build Status](https://travis-ci.org/CaruskiApps/EatOutLog.svg?branch=master)](https://travis-ci.org/CaruskiApps/EatOutLog)
+
+beta: [![Build Status](https://travis-ci.org/CaruskiApps/EatOutLog.svg?branch=beta)](https://travis-ci.org/CaruskiApps/EatOutLog)
+
+develop: [![Build Status](https://travis-ci.org/CaruskiApps/EatOutLog.svg?branch=develop)](https://travis-ci.org/CaruskiApps/EatOutLog) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/6ecd640348384369a9d7310fb03b1bb5)](https://www.codacy.com/app/ward486/EatOutLog?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=CaruskiApps/EatOutLog&amp;utm_campaign=Badge_Grade)
+
+##### Feature Ticket Tracking
+
+The tracking board can be found at: https://waffle.io/CaruskiApps/EatOutLog
+
+[![Stories in Ready](https://badge.waffle.io/CaruskiApps/EatOutLog.svg?label=ready&title=Ready)](http://waffle.io/CaruskiApps/EatOutLog)
```

Comment:
I'm not sure if I've quite figured out this badge, it shows 0 even though I put a dummy issue in 'ready'